contracts/SSVNetwork.sol

```diff
@@ -687,37 +545,49 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         _transferOperatorBalanceUnsafe(operatorId, operatorBalance.expand());
     }
 
-    /*
-    function withdrawPodBalance(bytes32 clusterId, uint256 amount) external override {
-        _validateClusterId(clusterId);
+    function withdrawPodBalance(
+        uint64[] memory operatorIds,
+        uint256 amount,
+        Pod memory pod
+    ) external override {
+        _validatePodIsNotLiquidated(pod);
```

Comment:
If we start to allow deposit to liquidated pod then we must remove this validation